samcli/lib/remote_invoke/stepfunctions_invoke_executors.py

```diff
@@ -0,0 +1,119 @@
+import logging
+import time
+from typing import Any
+
+from botocore.exceptions import ClientError, ParamValidationError
+
+from samcli.lib.remote_invoke.exceptions import (
+    ErrorBotoApiCallException,
+    InvalideBotoResponseException,
+    InvalidResourceBotoParameterException,
+)
+from samcli.lib.remote_invoke.remote_invoke_executors import (
+    BotoActionExecutor,
+    RemoteInvokeIterableResponseType,
+    RemoteInvokeLogOutput,
+    RemoteInvokeOutputFormat,
+    RemoteInvokeRequestResponseMapper,
+    RemoteInvokeResponse,
+)
+
+LOG = logging.getLogger(__name__)
+STATE_MACHINE_ARN = "stateMachineArn"
+EXECUTION_ARN = "executionArn"
+INPUT = "input"
+RUNNING = "RUNNING"
+SFN_EXECUTION_WAIT_TIME = 2
+
+
+class StepFunctionsStartExecutionExecutor(BotoActionExecutor):
+    """
+    Calls "start_execution" method of "stepfunctions" service with given input.
+    If a file location provided, the file handle will be passed as Payload object
+    """
+
+    _stepfunctions_client: Any
```

Comment:
Would suggest to use specific type rather than `Any`. We have `boto3-stub` as dev dependency, so we should use it to provide more accurate type hint.